packages/pds-ember/app/styles/pds/resets/index.scss

```diff
@@ -2,4 +2,5 @@
 @use "document";
 //@use "inert"; // uncomment when needed within PDS
 
-@use "../core/typography/reset";
+// Core Resets
```

Comment:
This comment will make more sense with part 2, where we add additional component resets.